src/interface/common.thrift

```diff
@@ -146,6 +147,10 @@ struct DataSet {
     2: list<Row>       rows;
 } (cpp.type = "nebula::DataSet")
 
+struct Geography {
+    1: string          wkb;
```

Comment:
How client handle the raw bytes?

Reply:
Currently, we can only print wkb directly. In the next pr, I will consider changing the Geography value to store the `Geometry`.